ui/lib/replication/addon/components/replication-summary.js

```diff
@@ -38,7 +38,9 @@ export default Component.extend(ReplicationActions, DEFAULTS, {
   initialReplicationMode: null,
   cluster: null,
 
-  replicationAttrs: alias('cluster.replicationAttrs'),
```

Comment:
wow, i totally forgot about `alias` ever being a thing. it's like time traveling! 😛